CHANGELOG.md

```diff
@@ -26,6 +26,12 @@
 
 ### Performance
 
+## 29.3.2
```

Comment:
just stick it under `main` up top - I'll deal with the versioning when releasing a new version 🙂

Reply:
done